systems/@PolynomialSystem/sampledFiniteTimeVerification.m

```diff
@@ -36,6 +36,7 @@
 % Implements the algorithm described in http://arxiv.org/pdf/1010.3013v1
 
 checkDependency('sedumi');
+checkDependency('distcomp'); % note: to operate without this, simply change the parfor to for in the loop below
```

Comment:
This shouldn't actually be necessary. I can't find it in the docs, but my recollection (supported by comments online) is that parfor will still run, albeit serially, on systems without PCT.